examples/loaders/panda_example.py

```
@@ -0,0 +1,60 @@
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+
+
+import logging
+
+import pandas as pd
+
+from camel.loaders import PandaReader
+
+logging.disable(logging.ERROR)
```

if we don't ignore the error we would get 
```
2025-01-10 22:03:12,521 - pandasai.helpers.logger - ERROR - Failed with error: Traceback (most recent call last):
  File "/Users/enrei/Library/Caches/pypoetry/virtualenvs/camel-ai-73PBX6Dc-py3.12/lib/python3.12/site-packages/pandasai/pipelines/chat/code_execution.py", line 85, in execute
    result = self.execute_code(code_to_run, code_context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/enrei/Library/Caches/pypoetry/virtualenvs/camel-ai-73PBX6Dc-py3.12/lib/python3.12/site-packages/pandasai/pipelines/chat/code_execution.py", line 174, in execute_code
    exec(code, environment)
  File "<string>", line 11, in <module>
  File "/Users/enrei/Library/Caches/pypoetry/virtualenvs/camel-ai-73PBX6Dc-py3.12/lib/python3.12/site-packages/pandasai/safe_libs/restricted_matplotlib.py", line 75, in __getattr__
    raise AttributeError(f"'{name}' is not allowed in RestrictedMatplotlib")
AttributeError: 'tight_layout' is not allowed in RestrictedMatplotlib
```

any insight for this? @liuxukun2000